extensions/wikia/Blogs/BlogArticle.php

```diff
@@ -83,8 +88,7 @@ private function showBlogListing() {
 		/**
 		 * use cache or skip cache when action=purge
 		 */
-		$user    = $this->mTitle->getBaseText();
-		$userMem = $this->mTitle->getPrefixedDBkey();
+		$owner = $this->getBlogOwner();
```

Comment:
So ```mTitle``` here will be a user's blog page (User_blog:Garthwebb), or a post (User_blog:Garthwebb/my_fantastic_post) and ```getBaseText``` returns the base title, e.g. no namespace and no subpage, i.e., 'Garthwebb'.  This is why I created ```getBlogOwner``` to make this a bit more clear.

The ```getPrefixedDBkey``` was misguided update in 2009 to (maybe?) be unecessarily specific by adding the namespace portion ```User_blog```.  However this method returns the subpage which means that when the ```clearBlogListing``` method below tried to clear this cache, it was trying to clear the "User_blog:Garthwebb/my_fantastic_post" URL which is not the "User_blog:Garthwebb" blog listing URL that the listing was originally cached under.

This simplifies things and fixes the caching issues.